### PR TITLE
Speech filter fixes

### LINKED ISSUE #30726: Genetics wordfilter issues
(WEB REPORT BY: kanef REMOTE: 206.221.180.138:7777)
# Revision
a8b0b98965c75ef83e623666d7513b7fa5c58ef0

# Description
Chav block is erroneously applied to multiple blocks, swedish block does not work
# Steps to Reproduce
Toggle on SWEDISH, ELVIS, CHAV and SMILE blocks seperately in player panel
# What you Expected
Seperate filters
# What Actually Happened
CHAV stacking on them, innit

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[bugfix][tested]

## What this does
Makes swedish wordfilter actuallly activate, lizard/insect/swedish wordfilters work with capitalization better.

## Why it's good
Closes #30726.

## Changelog
:cl:
 * bugfix: Swedish disability now works.
 * bugfix: Insectoids now properly yell their Zs

### CODE REVIEW

File: code/game/dna/genes/speech.dm [hunk 1]
```diff
@@ -225,7 +232,7 @@
 	addWordReplacement("beer","water with ice")
 	addWordReplacement("drink","water")
 	addWordReplacement("\\b(feminist|feminazi|SJW|social justice warrior)","empowered woman")
-	addWordReplacement("\\b(fuck you|i hate you|kill yourself|go die)","you're mean")
+	addWordReplacement("\\b(fuck you|i hate you|hug yourself|go die)","you're mean") //kill becomes hug
```
Comment (Exxion) [inline code/game/dna/genes/speech.dm]: I don't think this does what you meant

Comment (SECBATON-GRIFFON) [reply code/game/dna/genes/speech.dm]: eh, moving it before "kill" is filtered instead of changing it afterwards works too

Comment (Exxion) [reply code/game/dna/genes/speech.dm]: Ah, I see
Yeah, that's definitely a better solution

**Comment by Eneocho** [comment]
Does this fix the lisp having a lowercase h on the "Th" when screaming, or calling for lisps on weird locations, like saying Thhoe instead of "shoe" or the double thth when doing that "ss" thing as a skelemen?
See #29128, #29242 and #30208.
Also, that one request of not calling for lisps on non-verbal languages, such as golem, slime or clatter.

**Pull request merged**